lib/controllers/root-controller.js

```diff
@@ -426,8 +311,47 @@ export default class RootController extends React.Component {
     );
   }
 
+  renderFilePatches() {
+    if (this.props.filePatchItems.length === 0) {
+      return null;
+    }
+
+    return this.props.filePatchItems.map(item => {
+      const {filePath, stagingStatus, workdir} = getPropsFromUri(item.uri);
+
+      return (
+        <PaneItem
+          key={item.key}
+          workspace={this.props.workspace}
+          getItem={({subtree}) => subtree}
+          onDidCloseItem={() => { this.props.removeFilePatch(item); }}
```

Comment:
Maybe worth extracting into a method. The function is passed the PaneItem (but in this case will be the `StubItem`, so we'd have to get the real item).